### PR TITLE
Correct highlighting when colour is enabled. Rename logback.xml

### PR DESCRIPTION
**Pull request by rnc**
@thescouser89 No tests yet but I think this solves the highlighting issue...

### CODE REVIEW

**Comment by thescouser89** [comment]
Yup can confirm! Cloned your branch and it seems to work. Woah you did it!!!

File: cli/src/test/java/org/jboss/pnc/bacon/cli/AppTest.java [hunk 1]
```diff
@@ -30,6 +34,32 @@ void testHelp() throws Exception {
...
+                                            "-p",
+                                            configYaml.toString(),
+                                            "-v",
+                                            "pnc",
+                                            "-o",
+                                            "build",
+                                            "get",
+                                            "0" })));
+            assertThat(text).contains("Reconfiguring logger for NO_COLOR");
+            assertThat(text).contains(ANSIConstants.ESC_START);
```
[the remark this answers is not in the document]
Comment (rnc) [reply cli/src/test/java/org/jboss/pnc/bacon/cli/AppTest.java:57]: Its a little bit subtle and I had to remind myself as well :-) So, in verbose mode, the first two lines will have these characters until its reset, hence the test on line 58 checking that none of the following have it. Essentially it contains:
```
[[39mDEBUG[0;39m] - Log level set to DEBUG
[[39mDEBUG[0;39m] - Reconfiguring logger for NO_COLOR
[DEBUG] - Config file set from flag with profile default to /home/rnc/Work/ProjectNCL/bacon/pnc/target/test-classes/config.yaml
[DEBUG] - JSON command is enabled: true
```
I'll add another test without verbose to demonstrate that.

[the remark this answers is not in the document]
Comment (thescouser89) [reply cli/src/test/java/org/jboss/pnc/bacon/cli/AppTest.java:59]: Ah gotcha!

**Comment by thescouser89** [comment]
Thanks Nick! :) LGTM!

**Pull request merged**